node/repo/memrepo.go

```diff
@@ -11,13 +11,14 @@ import (
 	"github.com/ipfs/go-datastore"
 	"github.com/ipfs/go-datastore/namespace"
 	dssync "github.com/ipfs/go-datastore/sync"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
```

Comment:
Pretty sure the blockstore interface is aliased in lib/blockstore so we can only import one package here